<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server_Widget</class>
 <widget class="QWidget" name="Server_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QGroupBox" name="aj_core_connection_grpbx">
     <property name="title">
      <string>AJ-Core Verbindung</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="coreIP_lbl">
        <property name="text">
         <string>Core-IP:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="corePort_lbl">
        <property name="text">
         <string>Core-Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="corePWD_lbl">
        <property name="text">
         <string>Core-Passwort:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="coreConnect_btn">
        <property name="text">
         <string>Verbinden</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="corePWD_le"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="corePort_le"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="coreIP_le"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QGroupBox" name="new_server_grpbx">
     <property name="title">
      <string>Neuer Server</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="servername_lbl">
        <property name="text">
         <string>Servername:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="serverport_lbl">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="serverport_le"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="servername_le"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="addServer_btn">
        <property name="text">
         <string>Hinzuf√ºgen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Serverliste</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4" colspan="4">
    <widget class="QTableWidget" name="server_tbl">
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Servername</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP-Adresse</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Port</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>letzte Verbindung</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Verbindungsversuche</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0" rowspan="4" colspan="4">
    <widget class="QTabWidget" name="information_tab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Serverinfo</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Log</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
